src/lib/circuit_value.ts

```diff
@@ -77,8 +77,8 @@ abstract class CircuitValue {
     Circuit.assertEqual(this, x);
   }
 
-  isConstant(x: this) {
-    return x.toFields().every((x) => x.isConstant());
+  toConstant(): this {
```

Comment:
wait, it confuses me that you changed `isConstant` to `toConstant`. don't we want both?

Reply:
My bad, I got confused here. I reverted the change. :sweat_smile: